docs/pages/push-notifications/receiving-notifications.mdx

```diff
@@ -34,6 +36,180 @@ useEffect(() => {
 }, []);
 ```
 
+<Collapsible summary="Android Notification Object Example from addNotificationReceivedListener">
+
+Here's a sample of the notification object received by the callback function when using `Notifications.addNotificationReceivedListener`:
+
+```jsx
+console.log(notification);
+{
+  "request": {
+    "trigger": {
+      "remoteMessage": {
+        "originalPriority": 2,
+        "sentTime": 1724782348210,
+        "notification": {
+          "usesDefaultVibrateSettings": false,
+          "color": null,
+          "channelId": null,
+          "visibility": null,
+          "sound": null,
+          "tag": null,
+          "bodyLocalizationArgs": null,
+          "imageUrl": null,
+          "title": "Chat App",
+          "ticker": null,
+          "eventTime": null,
+          "body": "New message from John Doe",
+          "titleLocalizationKey": null,
+          "notificationPriority": null,
+          "icon": null,
+          "usesDefaultLightSettings": false,
+          "sticky": false,
+          "link": null,
+          "titleLocalizationArgs": null,
+          "bodyLocalizationKey": null,
+          "usesDefaultSound": false,
+          "clickAction": null,
+          "localOnly": false,
+          "lightSettings": null,
+          "notificationCount": null
+        },
+        "data": {
+          "channelId": "default",
+          "message": "New message from John Doe",
+          "title": "Chat App",
+          "body": "{\"senderId\":\"user123\",\"senderName\":\"John Doe\",\"messageId\":\"msg789\",\"conversationId\":\"conversation-456\",\"messageType\":\"text\",\"timestamp\":1724766427}",
+          "scopeKey": "@betoatexpo/expo-notifications-app",
+          "experienceId": "@betoatexpo/expo-notifications-app",
+          "projectId": "51092087-87a4-4b12-8008-145625477434"
+        },
+        "to": null,
+        "ttl": 0,
+        "collapseKey": "dev.expo.notificationsapp",
+        "messageType": null,
+        "priority": 2,
+        "from": "115310547649",
+        "messageId": "0:1724782348220771%0f02879c0f02879c"
+      },
+      "channelId": "default",
+      "type": "push"
+    },
+    "content": {
+      "autoDismiss": true,
+      "title": "Chat App",
+      "badge": null,
+      "sticky": false,
+      "sound": "default",
+      "body": "New message from John Doe",
+      "subtitle": null,
+      "data": {
+        "senderId": "user123",
+        "senderName": "John Doe",
+        "messageId": "msg789",
+        "conversationId": "conversation-456",
+        "messageType": "text",
+        "timestamp": 1724766427
+      }
+    },
+    "identifier": "0:1724782348220771%0f02879c0f02879c"
+  },
+  "date": 1724782348210
+}
+
+// You can access the notification custom data as follows:
```

Comment:
````suggestion
You can directly access the notification custom data by logging the `notification.request.content.data` object:

```json
````

It's always best to describe a code snippet in words so that people can read it or it grabs their attention. Often times people prefer to skim and important info can be easily missed out by them.